pkg/provision/storage/perWorkspaceStorage.go

```diff
@@ -189,6 +199,15 @@ func getPVCSize(workspace *common.DevWorkspaceWithConfig, namespacedConfig *nsco
 			}
 			requiredPVCSize.Add(volumeSize)
 		}
+		if component.Container != nil {
+			if component.Container.MountSources != nil && *component.Container.MountSources {
```

Comment:
Should we use https://github.com/devfile/devworkspace-operator/blob/9325f55c9d23d44d6ceda181e91043b51e7fa73d/pkg/library/container/mountSources.go#L34 ?

Reply:
Good catch ! Yes we should.